### ISSUE TITLE
Update required

### ISSUE THREAD

**Issue by ghost** [divrhino/hugotails#7]
Hi @divrhino 

this does not work any more with tailwind version 3.0.+

an update to this will be great

**Comment by pbgswd** [comment]
Yes there have been some changes to Tailwind! It sounds like the Hugo team is working on it but it is a challenge. 

Have a look at these links:

A discussion of a workaround: https://www.brycewray.com/posts/2021/11/making-tailwind-jit-work-hugo/ 

A repo where it works:
https://github.com/praveenjuge/hugo-tailwind-jit
